prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts

```diff
@@ -7,11 +14,28 @@ export enum AgreementType {
   PHARMACY_OBO_TOA
 }
 
+
 export const AgreementTypeNameMap: Record<AgreementType, string> = {
-  [AgreementType.COMMUNITY_PHARMACIST_TOA]: 'Pharma Regulated Users',
-  [AgreementType.REGULATED_USER_TOA]: 'Regulated Users',
-  [AgreementType.OBO_TOA]: 'On Behalf Of',
-  [AgreementType.COMMUNITY_PRACTICE_ORGANIZATION_AGREEMENT]: 'Community Practice Organization',
-  [AgreementType.COMMUNITY_PHARMACY_ORGANIZATION_AGREEMENT]: 'Community Pharmacy Organization',
-  [AgreementType.PHARMACY_OBO_TOA]: 'Pharma On Behalf Of'
+  [AgreementType.COMMUNITY_PHARMACIST_TOA]: COMMUNITY_PHARMACIST_TOA,
+  [AgreementType.REGULATED_USER_TOA]: REGULATED_USER_TOA,
+  [AgreementType.COMMUNITY_PRACTICE_ORGANIZATION_AGREEMENT]: COMMUNITY_PRACTICE_ORGANIZATION_AGREEMENT,
+  [AgreementType.COMMUNITY_PHARMACY_ORGANIZATION_AGREEMENT]: COMMUNITY_PHARMACY_ORGANIZATION_AGREEMENT,
+  [AgreementType.OBO_TOA]: OBO_TOA,
+  [AgreementType.PHARMACY_OBO_TOA]: PHARMACY_OBO_TOA
 };
+
+// Paper Enrolment
+export enum AgreementTypePaperEnrolment {
+  COMMUNITY_PHARMACIST_TOA = 1,
+  REGULATED_USER_TOA,
+  OBO_TOA,
+  PHARMACY_OBO_TOA
+}
+
+export const AgreementTypeNameMapPaperEnrolment: Record<AgreementTypePaperEnrolment, string> = {
```

Comment:
I know this isn't really defined anywhere, but probably want to put the description of the variable as a Prefix with the type of the object being the suffix.

So like PaperEnrolmentAgreementTypeNameMap

Reply:
Got it, thanks!